rnndb/falcon.xml

```diff
@@ -298,6 +300,9 @@
 		<bitfield pos="30" name="UAS" variants="GF119-"/> <!-- unified address space -->
 		<bitfield pos="31" name="UNK31" variants="GF119-"/> <!-- auto fill -->
 	</reg32>
+
+	<reg32 offset="0x130" name="UC_CTRL_ALIAS" type="fecs_cpuctl" variants="GM107-"/> <!-- v5+ secretful? -->
```

Comment:
Sounds strange -- any idea how it differs from 0x100?

Reply:
I've observed nvgpu and blob utilize this alias UC_CTRL register, only whilst in Heavy Secure mode.
It appears important given their behavior to track usage of both registers.